frontend/src/Pages/LoginSuccess.js

```diff
@@ -20,10 +20,12 @@ export default function LoginSuccess() {
                 type: "m.login.token",
                 token: loginToken
             });
-            authenticateWithOpenId(response.data);
+            await authenticateWithOpenId(response.data);
```

Comment:
```suggestion
```

Reply:
I'm actually still using `authenticateWithOpenId` since I'll need it for both types of logins (password and sso).
The matrix API only takes care of fetching the Open ID token. And this method takes care of authenticating with our backend server.